src/database/migrations/0026-alter-base_fields-add-field_scope.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TYPE field_scope AS ENUM ('proposal', 'organization');
```

Comment:
A little documentation here or in the other definition of the same enum in TypeScript would be helpful. If I recall correctly, these scopes are not always mutually exclusive. For example, an organization-scoped field can be submitted as part of a proposal, right? But the converse is not necessarily true.

Reply:
Good call -- I wonder if SQL lets you add a comment for a specific enum value...